### ISSUE TITLE
[Bug]: Slider handler is not keyboard accessible when the slider has `editable` attribute

### ISSUE THREAD

**Issue by Rocss** [adobe/spectrum-web-components#4906]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-slider

### Expected behavior

Both the slider and the input should be keyboard focusable and accessible.

### Actual behavior

The handler is keyboard accessible only when the slider is not `editable`

### Screenshots

N/A

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/slider/
2. Navigate using the keyboard
3. Observe there is no tab stop on the handler when the slider is editable

### Sample code or abstract reproduction which illustrates the problem

Seems like this was intentional, as per https://github.com/adobe/spectrum-web-components/blob/bd35e1b7763675fae79e52ef88cdaa661c3fcdeb/packages/slider/src/HandleController.ts#L557

### Severity

None

**Comment by jnurthen** [comment]
The intent behind this was to reduce the number of tab stops when interacting with a page - particularly one with lots of sliders as is common on photo editing applications. All of the functionality of the slider can be accomplished by interacting directly with the edit field directly and the slider will echo those changes so the slider was seen as being redundant in this case.

I'm very interested to know if this is something that was reported by a user who couldn't operate the control or if it was something reported by someone performing testing.

**Comment by najikahalsema** [comment]
@Rocss could you provide some context for @jnurthen above?

**Comment by Rocss** [comment]
@najikahalsema I already responded on the slack thread: https://adobe.slack.com/archives/CESK60MQD/p1730713316574589
